service/routes.py

```diff
@@ -6,6 +6,7 @@
 
 from flask import jsonify, request, url_for, make_response, abort
 from service.common import status  # HTTP Status Codes
+# pylint: disable=cyclic-import
```

Comment:
This error was occuring in the test_routes.py file, so can you place this line over there?

Reply:
No, that wouldn't work. I tried. See prof's response to this issue in slack

Reply:
Ok perfect, I am merging this.